livechat/utils/ws_client.py

```diff
@@ -110,7 +111,7 @@ def _keep_ws_alive(self) -> None:
                     )
                     break
                 else:
-                    self.logger.info('Ping action sent.')
+                    self.logger.debug('Ping action sent.')
```

Comment:
debug instead of info?

Reply:
yeah, I've changed this after discussion with @DamianAmbrozkiewicz  - we don't want to see this information by default as it keeps on popping up when e.g. using our sdk in interactive shell. Do you disagree with that?